python/pfs/instmodel/detector.py

```diff
@@ -94,7 +99,10 @@ def saturate(self, exp):
     
     def readout(self, exp, flux, exptime=1.0, ontoBias=None,
                 applyFlat=None):
-        """ 'Readout' an exposure: add bad columns, hot pixels, etc. """
+        """ 'Readout' an exposure: add bad columns, hot pixels, etc.
```

Comment:
Minor: Should there be a whitespace after the triple-quote? I guess it maybe necessary to make the single-quoted word clear, but please check and confirm.

Reply:
It's the style throughout this file. Following the current style is more important than following the correct style.